Framework/Core/include/Framework/DataSamplingPolicy.h

```diff
@@ -36,39 +37,19 @@ namespace framework
 class DataSamplingPolicy
 {
  private:
-  // todo: see if dpl matchers can be used here instead of this strange construction
-  struct inputSpecHasher {
-    size_t operator()(const InputSpec& i) const
+  using V = std::vector<std::pair<InputSpec, OutputSpec>>;
```

Comment:
CodingGuidelines specify that names should be "meaningful; abbreviations should be avoided" . Do you think that there is a more meaningful name for `V` ? if not I will let it go.

Reply:
Hmm, maybe `PathVectorBase`?

Reply:
ok

Reply:
done